library.properties

```diff
@@ -1,5 +1,5 @@
 name=OXRS-SHA-LCD-ESP32-LIB
-version=0.1.1
+version=0.2.1
```

Comment:
When you increment the minor version, reset the revision to zero - i.e. this should be `0.2.0`